### DISCUSSION TITLE
デザインについて

### DISCUSSION THREAD

**Discussion by tkoyama010** [FEABeRjp/FEABeR#6] [General]
決めること
- [ ] プロジェクトのロゴ
- [x] ページの構成

**Comment by tkoyama010** [comment]
プロジェクトのロゴはデザイナーに依頼中です。

**Reply by mmer547** [reply]
素案来たのでDMで送ります

**Reply by mmer547** [reply]
ロゴはsvg形式の方がいいでしょうか？

**Reply by tkoyama010** [reply]
後で編集できるのでsvgでお願いいたします。必要に応じて他の形式に変換いたします。

**Reply by mmer547** [reply]
承知しました。svgで上げるようにいたします

**Comment by tkoyama010** [comment]
ページの構成は[CoFEA](https://cofea-ja.readthedocs.io/ja/latest/)プロジェクトを参考にすると良さそうです。[機械学習帳](https://chokkan.github.io/mlnote/)もデザインの参考になりそうです。

**Reply by mmer547** [reply]
1ページ1問題がいいかなと思っています

**Reply by mmer547** [reply]
コンテンツ群（サイト説明、ブログリンクなど）の下階層に問題結果のページがあって、ページ構成ですけど計算の分野（構造、振動、熱、他）があって、その下に問題という構成かなと。問題集ベースで行くより、現象の分野で割り振った方が見る側はわかりやすいかと思いました

**Reply by tkoyama010** [reply]
この構成ですね。私もこれで良いと思います。最初に構成だけ作っておきます。
```
コンテンツ群（サイト説明、ブログリンクなど）
└─ 問題結果
      ├─構造
      │  ├─ 問題1
      │  └─ 問題2
      ├─振動
      │  ├─ 問題1
      │  └─ 問題2
      ├─熱
      │  ├─ 問題1
      │  └─ 問題2
      └─他
```

**Reply by mmer547** [reply]
ありがとうございます
